src/test/java/com/gistpetition/api/acceptance/petition/ReleasePetitionAcceptanceTest.java

```diff
@@ -0,0 +1,76 @@
+package com.gistpetition.api.acceptance.petition;
+
+import com.gistpetition.api.acceptance.AcceptanceTest;
+import com.gistpetition.api.acceptance.common.TUser;
+import com.gistpetition.api.petition.domain.AgreementRepository;
+import com.gistpetition.api.petition.domain.Category;
+import com.gistpetition.api.petition.domain.PetitionRepository;
+import com.gistpetition.api.petition.dto.AgreementRequest;
+import com.gistpetition.api.petition.dto.PetitionRequest;
+import com.gistpetition.api.petition.dto.TempPetitionResponse;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+
+import java.util.List;
+
+import static com.gistpetition.api.acceptance.common.TUser.*;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ReleasePetitionAcceptanceTest extends AcceptanceTest {
+
+    @Autowired
+    PetitionRepository petitionRepository;
+
+    @Autowired
+    AgreementRepository agreementRepository;
+
+    @Test
+    @DisplayName("청원을 만들고 5명이 동의해서 Manager가 release하는 테스트")
+    void ReleasePetition() {
```

Comment:
```suggestion
    void releasePetition() {
```
_**method name start with small letter**_